rlImGui.cpp

```diff
@@ -99,7 +99,11 @@ static void rlImGuiNewFrame(float deltaTime)
     }
     else 
     {
+#if __APPLE__
+        io.DisplayFramebufferScale = ImVec2(2.0f, 2.0f);
+#else
         io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f);
+#endif
```

Comment:
I don't think this is correct. You can't just assume every apple computer is running at a buffer scale of 2x, Not all devices are retina.
You would need some OS specific code to find the actual display scale and set it.
I can't accept this as it may break non-retina macs.